manifests/t/TheWaWaR/simple-http-server/0.6.10/TheWaWaR.simple-http-server.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: TheWaWaR.simple-http-server
+PackageVersion: 0.6.10

```

```suggestion
PackageVersion: 0.6.9
```
Please update file path to match this change.